### DISCUSSION TITLE
port to mobaxterm

### DISCUSSION THREAD

**Discussion by minhtuan1407** [daltonmenezes/aura-theme#113] [Port request]
Love your theme, hope that you can port aura theme to mobaxterm

**Comment by im-coder-lg** [comment]
Now, I use MobaXTerm and I have kinda fiddled with its settings. There is actually no way to port any other theme than the preset ones on MobaXTerm. If you do know how to do this, then list the steps below, and the files needed. I will try porting for MobaXTerm but one side of me tells it's kinda pointless.
